### PR TITLE
Adding the rest of the basic moderation commands

### LINKED ISSUE #8: Basic moderation commands
### General Troubleshooting

- [X] I have checked for similar issues on the Issue-tracker.
- [X] I have checked the branches or the maintainers' PRs for upcoming features.

### Feature Request

Many commands for moderate users :

- [x] Ban/Unban
- [x] Tempban
- [x] Kick
- [x] TimeOut/Remove Timeout
- [x] Warn/remove warn
- [x] See list of infractions
- [x] Delete list of message of one user
- [x] Delete an amount of message in a channel

### Example Use-Case

_No response_

### PR DESCRIPTION
**Pull request by RealYusufIsmail**
[code_of_conduct]: https://github.com/Programming-Org/Programming/blob/master/.github/CODE_OF_CONDUCT.md
[new_issue]: https://github.com/Programming-Org/Programming/issues/new/choose

## Pull-request

- [x] I have read the [code of conduct][code_of_conduct].
- [x] I have created a relating [issue][new_issue].

### Changes

- [x] Existing code
- [x] New feature

Closes : #8 

## Description

I have added all the commands list in issue #8, this also means that there will be new additions to the database

### CODE REVIEW

File: src/main/resources/sql/tenpban.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS tempban
+(
+    id           serial       not null primary key,
+    guild_id     varchar(255) not null,
+    user_id      varchar(255) not null,
+    moderator_id varchar(255) not null,
+    duration     interval         not null,
+    start        timestamp    not null,
+    reason       varchar(255) not null
+)
```
Comment (Haleloch) [inline src/main/resources/sql/tenpban.sql]: Typo in file name

File: src/main/java/io/github/org/programming/bot/config/LoggerColorScheme.java [hunk 2]
```diff
@@ -1,51 +0,0 @@
...
-        return switch (event.getLevel().toInt()) {
-            case Level.DEBUG_INT -> ANSIConstants.BLUE_FG;
-            case Level.INFO_INT -> ANSIConstants.GREEN_FG;
-            case Level.WARN_INT -> ANSIConstants.YELLOW_FG;
-            case Level.ERROR_INT -> ANSIConstants.RED_FG;
-            case Level.TRACE_INT -> ANSIConstants.CYAN_FG;
-            default -> ANSIConstants.DEFAULT_FG;
-        };
-    }
-}
```
Comment (Haleloch) [inline src/main/java/io/github/org/programming/bot/config/LoggerColorScheme.java]: why did you remove the colors?

File: src/main/java/io/github/org/programming/database/Database.java [hunk 3]
```diff
@@ -61,11 +62,12 @@ private static void executeTableUpdate() {
 
             logger.info("Executing table updates...");
 
+            assert listOfFiles != null;
             for (File file : listOfFiles) {
                 if (file.isFile()) {
                     logger.info("Executing update: {}", file.getName());
```
Comment (Haleloch) [inline src/main/java/io/github/org/programming/database/Database.java]: What do you want to log ?

**Review by Haleloch — CHANGES_REQUESTED** [review]

File: src/main/resources/sql/tempban.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS tempban
+(
+    id           serial       not null primary key,
+    guild_id     varchar(255) not null,
+    user_id      varchar(255) not null,
+    moderator_id varchar(255) not null,
+    duration     interval         not null,
```
Comment (RealYusufIsmail) [inline src/main/resources/sql/tempban.sql]: Need to align

File: src/main/resources/sql/moderation.sql [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
-    moderator_id varchar(255) not null,
-    time_stamp   timestamp    not null default current_timestamp,
-    reason       varchar(255) not null,
-    type         varchar(255) not null
+    id                 serial       not null primary key,
+    guild_id           varchar(255) not null,
+    user_id            varchar(255) not null,
+    moderator_id       varchar(255) not null,
+    time_stamp         timestamp    not null default current_timestamp,
+    amount_of_warnings int          null,
```
Comment (RealYusufIsmail) [inline src/main/resources/sql/moderation.sql:8]: Align

File: src/main/resources/logback.xml [hunk 6]
```diff
@@ -1,21 +1,32 @@
...
-    <root level="error">
-        <appender-ref ref="CONSOLE"/>
+    <root level="${ROOT_LOG_LEVEL:-WARN}">
+        <appender-ref ref="STDOUT"/>
+        <appender-ref ref="LOGFILE"/>
     </root>
-
 </configuration>
+
+
```
Comment (RealYusufIsmail) [inline src/main/resources/logback.xml]: Need to replace with updated version

File: src/main/resources/sql/tempban.sql (hunk 4 above)
Comment (Haleloch) [reply src/main/resources/sql/tempban.sql]: Solve ?

File: src/main/resources/sql/moderation.sql (hunk 5 above)
Comment (Haleloch) [reply src/main/resources/sql/moderation.sql:8]: Solve ?

File: src/main/resources/logback.xml (hunk 6 above)
Comment (Haleloch) [reply src/main/resources/logback.xml]: ??

File: src/main/jooq/io/github/org/programming/jooq/tables/records/TempbanRecord.java [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+            setId(value.getId());
+            setGuildId(value.getGuildId());
+            setUserId(value.getUserId());
+            setModeratorId(value.getModeratorId());
+            setDuration(value.getDuration());
+            setStart(value.getStart());
+            setReason(value.getReason());
+        }
+    }
+}
```
Comment (Haleloch) [inline src/main/jooq/io/github/org/programming/jooq/tables/records/TempbanRecord.java:356]: What is the goal of this file ?

File: src/main/jooq/io/github/org/programming/jooq/tables/Moderation.java [hunk 8]
```diff
@@ -15,7 +15,7 @@
 import org.jooq.Identity;
 import org.jooq.Name;
 import org.jooq.Record;
-import org.jooq.Row6;
+import org.jooq.Row8;
```
Comment (Haleloch) [inline src/main/jooq/io/github/org/programming/jooq/tables/Moderation.java:18]: What is Row6/Row8/Record8/... ?

File: src/main/jooq/io/github/org/programming/jooq/tables/records/TempbanRecord.java (hunk 7 above)
Comment (RealYusufIsmail) [reply src/main/jooq/io/github/org/programming/jooq/tables/records/TempbanRecord.java:356]: This is auto generated by jooq, i.e it basically gets the parameter's in the database file and translates it to code so i can use it in my code.

File: src/main/jooq/io/github/org/programming/jooq/tables/Moderation.java (hunk 8 above)
Comment (RealYusufIsmail) [reply src/main/jooq/io/github/org/programming/jooq/tables/Moderation.java:18]: No idea, its generated by jooq.

Comment (RealYusufIsmail) [reply src/main/jooq/io/github/org/programming/jooq/tables/Moderation.java:18]: https://www.jooq.org

**Review by Haleloch — APPROVED** [review]
OK

**Comment by RealYusufIsmail** [comment]
@all-contributors please add @RealYusufIsmail for infrastructure, tests and code

**Comment by RealYusufIsmail** [comment]
@all-contributors please add @Haleloch for code and review

**Comment by RealYusufIsmail** [comment]
@all-contributors please add @RealYusufIsmail for infrastructure, tests, review and code

**Comment by RealYusufIsmail** [comment]
https://github.com/all-contributors please add @Haleloch for code and review

**Comment by RealYusufIsmail** [comment]
@all-contributors please add @Haleloch for code and review

**Comment by RealYusufIsmail** [comment]
@all-contributors please add @Haleloch for code and review
